README.md

```diff
@@ -290,6 +290,44 @@ RPC options:
 ```
 <!-- END_CLI_HELP -->
 
+### Syncing with Holesky
+
+#### Step 1: Set up a jwt secret for both clients
+
+As an example, we put the secret in a `secrets` directory in the home folder.
+
+```bash
+sudo mkdir -p ~/secrets
```

Comment:
Is `sudo` really necessary here? If not, let's remove it

Reply:
Sure